randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -851,12 +861,18 @@ Extra - room_id: [25]
       Any of the following:
           Can Use Power Bombs
           All of the following:
-              Can Use Bombs
+              Can Activate Pillar and Can Use Bombs
```

Comment:
Why `Can Activate Pillar`? Bombs already activates pillars, no?

Reply:
they do, i just added it for the sake of being inuitive, willing to remove if you think otherwise

Reply:
oh wait nvm on this thats what comments are for